ls1intum/Apollon
# Bugfix/assessment dialog wrong order of subelements

### Checklist

### Motivation and Context


Solves https://github.com/ls1intum/Artemis/issues/1342

### Description
Container now reorders children, when children are appended to container. Each container implements a reoroderChidlren to do that 
I also removed some dead code appendElements and remove Elements from UMLContainer

### Steps for Testing


1. create class or object diagram
2. add methods and attributes in arbitrary order
3. make sure to add a method before another attribute
4. go to assessment view and check assessment popup of element
5. expected result: assessment items are in the correct order (attributes before methods and order within as depicted in diagram)


### Screenshots


![image](https://user-images.githubusercontent.com/22955066/81043827-5683f480-8eb3-11ea-9e86-a1486877f30c.png)
